airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -518,6 +528,11 @@ def run_query(self,
         :param use_legacy_sql: Whether to use legacy SQL (true) or standard SQL (false).
             If `None`, defaults to `self.use_legacy_sql`.
         :type use_legacy_sql: boolean
+        :param api_resource_configs: a dictionary that contain params
```

Comment:
NIT: Can you change the wording to say (something similar to) that you only need to use this if you want to pass parameters that are not already in this operators.

Reply:
Apart from this, it LGTM. 

@tswast Any final comments?

Reply:
@kaxil check it